query/src/main/java/io/keen/client/java/Query.java

```diff
@@ -36,8 +42,8 @@
      *
      * @return The JSON object map.
      */
-    public Map<String, Object> constructQueryArgs() {
-
+    @Override
+    Map<String, Object> constructRequestArgs() {
```

Comment:
We were generally trying to wrangle in the exposed surface area of the SDK in the parts we touched that we deemed probably should have been package internal to begin with, but yeah a breaking change is a breaking change. Do you have a strong opinion one way or the other on this specific change? It would imply making this a public method on the KeenQueryRequest abstract base.